lib/index.js

```diff
@@ -75,6 +80,16 @@ info: Storage engine shut down.\n";
       process.stderr.write(output);
     }
   });
+
+  self._server = server;
+}
+
+Server.prototype.stop = function(cb) {
+  var self = this;
+  self._server.close(cb);
```

Comment:
Does this work? If you close the server, aren't all the connections going to emit an end/error event before you have a chance to close them?